packages/suite/src/hooks/wallet/useCoinmarketSpend.ts

```diff
@@ -42,11 +43,15 @@ const useSpendState = ({ selectedAccount, fees }: Props, currentState: boolean)
 export const useCoinmarketSpend = (props: Props): SpendContextValues => {
     const [voucherSiteUrl, setVoucherSiteUrl] = useState<string | undefined>('error');
 
-    const { addNotification, setShowLeaveModal, saveTrade } = useActions({
+    const { addNotification, setShowLeaveModal, saveTrade, loadInvityData } = useActions({
         addNotification: notificationActions.addToast,
         setShowLeaveModal: coinmarketSellActions.setShowLeaveModal,
         saveTrade: coinmarketSpendActions.saveTrade,
+        loadInvityData: coinmarketCommonActions.loadInvityData,
     });
+
+    loadInvityData();
```

Comment:
`loadInvityData` will be called every time when `SpendLoaded` will be rendered, every time selectedAccount, device, lang, fees are changed  (i've counted at least 4 times at 1st load)

it should be wrapped in some memo or something, no? 
i noticed that other invity hooks are doing the same thing, calling `@coinmarket-common/load_data` multiple times

Reply:
Fixed.